### DISCUSSION TITLE
Einweisungsvorlage für alle offenen Einweisungen eines Mitarbeiters

### DISCUSSION THREAD

**Discussion by moritzvolkmar** [Samedis-care/samedis-care-community#4] [Idea retired / denied]
Aktuell müsste man die Matrix öffnen und entsprechend den MA und die Gruppe auswählen und parallel manuell einen Einweisungsvorgang anlegen und dort ebenfalls manuell entsprechende Typen auswählen.

Vorgang automatisieren? Etwas für my.samedis.care? Generell anbieten? 

Hängt zusammen mit: 

https://github.com/Samedis-care/samedis-care-issues/discussions/401

1. Search field in the matrix also search for device models? Currently searches only by employee
2. Offer more interactivity
- Click on MA -> briefing form directly from the matrix
- Click on Type -> List with all MA that have not yet been briefed
3. Underline/legend missing Devices and Employee Matrix

**Comment by YvesR** [comment]
We can do meanwhile create trainings directly out of the matrix report.
